src/views/LoginView.vue

```diff
@@ -136,25 +136,27 @@ onBeforeMount(() => {
 
 // TODO: error handling and error toasts
 // See: https://github.com/GEWIS/sudosos-frontend-vue3/issues/46
-const loginHandler = loginForm.handleSubmit(async () => {
-  if (username.value.modelValue.includes('@')) {
-    await apiService.authenticate.localAuthentication({
-      accountMail: username.value.modelValue,
-      password: password.value.modelValue }).then((res) => {
+const loginHandler = loginForm.handleSubmit(async (values) => {
+  try {
+    if (values.username.includes('@')) {
+      const res = await apiService.authenticate.localAuthentication({
```

Comment:
userstore should have localauth added for consistensy but this is not frontend related